src/server.cpp

```diff
@@ -1413,7 +1413,6 @@ CVector<CChannelInfo> CServer::CreateChannelList()
             // append channel ID, IP address and channel name to storing vectors
```

Comment:
"IP Address"?

Reply:
Thanks for the review, you've certainly got a point here. I pushed a new commit removing this comment altogether.